src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. You should remove the empty line before the class declaration.